courses/free-courses-en.md

```diff
@@ -1610,6 +1610,7 @@
 ### Scala
 
 * [Functional Programming in Scala Specialization](https://www.coursera.org/course/reactive)
+* [Scala Tutorial](https://www.youtube.com/playlist?list=PLmtsMNDRU0BxryRX4wiwrTZ661xcp6VPM) - Rock the JVM (Youtube)
```

Comment:
```suggestion
* [Scala at Light Speed](https://www.youtube.com/playlist?list=PLmtsMNDRU0BxryRX4wiwrTZ661xcp6VPM) - Rock the JVM
```
Don't invent titles! Use the title given by the resource.

Reply:
@eshellman thanks for pointing it out. updated. can u please re-review